.github/workflows/e2e-juju-machine-charm-candid-auth.yml

```diff
@@ -0,0 +1,34 @@
+name: "E2E tests: Juju, machine charm and local auth"
+
+on:
+  workflow_call:
+    inputs:
+      admin-password:
+        required: false
+        type: string
+        description: The password to use when logging in as the admin user.
+        default: password1
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        node-version: [22.x]
+    steps:
+      - name: Checkout Juju Dashboard repo
+        uses: actions/checkout@v4
+      - name: Set up candid
+        uses: ./.github/actions/setup-candid
+      - name: Set up access
+        run: echo ${{ inputs.admin-password }} | juju change-user-password --no-prompt
```

Comment:
I don't think we need to do this when using Candid as we're using the password set in the provider.